circle.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       # now install and checks the resulting file (just in case)
       - run:
           command: |
-            dnf install -y meson python3-devel pygobject3-devel
+            dnf install -y meson gettext python3-devel pygobject3-devel
```

Comment:
just squash this in with the Add localization support commit please, I think that's easier than a standalone commit that doesn't explain much. It's  just a single line here anyway